elastic/kibana
# [Synthetics] Overview - add search bar

## Summary

Resolves https://github.com/elastic/kibana/issues/142523
Relates to https://github.com/elastic/kibana/issues/135160

Added Overview search bar and quick filters for Up, Down, or Disabled status. 

<img width="1375" alt="Screen Shot 2022-11-07 at 1 59 42 PM" src="https://user-images.githubusercontent.com/11356435/200450640-1586d452-0e57-4477-886c-e60d0dd3d8d1.png">
<img width="1376" alt="Screen Shot 2022-11-07 at 1 59 10 PM" src="https://user-images.githubusercontent.com/11356435/200450644-ee2b1415-8cc3-4c44-9661-c6246390b5e9.png">
<img width="1387" alt="Screen Shot 2022-11-07 at 1 59 18 PM" src="https://user-images.githubusercontent.com/11356435/200450646-e3eed782-a143-4ad0-adbb-ea5f6b6e1d71.png">
<img width="1382" alt="Screen Shot 2022-11-07 at 1 59 26 PM" src="https://user-images.githubusercontent.com/11356435/200450647-821c8376-86dd-449d-919c-897d7946ac8c.png">

### Testing
1. Create a few different monitors, ensuring a mix of up and down monitors
2. Search for the monitor via the search bar. Ensure that the monitor appears
3. Filter by up and down status. Ensure the correct monitors appear
4. Sort the monitors while using the up and down filters. Ensure the monitors are filtered by status and sorted at the same time
5. Enter a search that matches no monitors. Confirm the no monitors found UX appears
